blocks/components/editable/index.js

```diff
@@ -233,7 +233,7 @@ export default class Editable extends wp.element.Component {
 		}
 
 		content = wp.element.renderToString( content );
-		this.editor.setContent( content );
+		this.editor.setContent( content, { format: 'raw' } );
```

Comment:
I can be convinced, but this adds a lot of things for us to do. Not only when merging but also when parsing. We should provide helpers maybe to do this filtering, because it can be a difficult task to do and we should leverage this in a unified way.